### ISSUE TITLE
Error when create stack: stack names may not contain slashes

### ISSUE THREAD

**Issue by ladung** [pulumi/pulumi#11415]
### What happened?

When I create stack with command:
```
  dungla@pop-os  /data/ladung/pulumi/dev  
╰─ pulumi stack init company/dev
error: stack names may not contain slashes
```
Why I do not create stack with ```orgName/stackName```

### Steps to reproduce

1

### Expected Behavior

1

### Actual Behavior

1

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by dixler** [comment]
Hi. Thanks for posting. I believe this error occurs if you use the filestate backend rather than the pulumi service (via `pulumi login`). Organizations are only supported on the Pulumi service.

If you have more questions or this doesn't solve your issue, definitely let us know! Thanks!

**Comment by nstires-ctgx** [comment]
I saw this consistently with one of my stacks in the previous version of pulumi CLI. The workaround I found was to pass the stack argument:

`pulumi up -s my-team/app-deploy/my-stack`

All of my stacks follow the same naming convention, and it was isolated to `my-team/app-deploy`

This does appear to be resolved in CLI v3.47.1.
